src/test/pd_test.cc

```diff
@@ -0,0 +1,49 @@
+#include <unistd.h>
+
+#include <pd/Client.h>
+#include <iostream>
+#include "MockPDServer.h"
+
+namespace pingcap
+{
+namespace test
+{
+
+bool testPDGetGCSafePoint()
+{
+    std::vector<std::string> addrs;
+    for (int i = 1; i <= 3; i++)
+    {
+        addrs.push_back("127.0.0.1:" + std::to_string(5000 + i));
+    }
+
+    PDServerHandler handler(addrs);
+
+    PDService * pd_server = handler.RunPDServer();
+
+    ::sleep(1);
```

Comment:
Consider `std::this_thread::sleep_for`, then unistd.h can be removed

Reply:
OK

Reply:
PTAL again

Reply:
PTAL again

Reply:
> PTAL again

Great work! But in `src/test/pd_test.cc` and `src/pd/Client.cc`, you still include `unistd.h`. Is there somewhere you're using it in these two files?

Reply:
fixed